run.cmd

```diff
@@ -0,0 +1,30 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+if not defined VisualStudioVersion (
```

Comment:
At some other point we might consider putting this logic in msbuild.cmd instead of run.cmd, something to consider later.